.github/workflows/php.yml

```diff
@@ -0,0 +1,21 @@
+name: github test
+
+on: [push]
+
+jobs:
```

Comment:
is the intention of this to replace Travis? I'm not opposed to that per se though all my other projects use Travis so I'm kind of inclined to stick with it - either way though I guess we only need / should have one CI

Reply:
I'll drop it; it'd just be an alternative if you ever wanted it.